tests/unit/models/transactions/test_credential_accept.py

```diff
@@ -0,0 +1,60 @@
+from unittest import TestCase
+
+from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.transactions.credential_accept import CredentialAccept
+from xrpl.utils import str_to_hex
+
+_ACCOUNT_ISSUER = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_ACCOUNT_SUBJECT = "rNdY9XDnQ4Dr1EgefwU3CBRuAjt3sAutGg"
```

Comment:
The account prefix is unnecessary here

Reply:
Outside the context of credentials, it is not immediately obvious if `Subject` refers to an "account". I prefer to retain the Account prefix.

Reply:
what do you think of `_ISSUER_ACCT` and `_SUBJECT_ACCT` ? Is that more informative?

Reply:
54d894db34fd90c7ff0fa6c6bf50bc4aef901def